main/tests.py

```diff
@@ -1,3 +1,84 @@
 from django.test import TestCase
+from .models import Role
+#
+from rest_framework.test import APIClient
+from rest_framework import status
+from django.core.urlresolvers import reverse
+#
+from django.contrib.auth.models import User
+
 
 # Create your tests here.
+
+
+class ModelTestCase(TestCase):
+    """Defines the test suite for the ROLE model."""
+
+    def setUp(self):
+        user = User.objects.create(username='nerd')
+        self.role_name = 'editor'
+        self.role = Role(name=self.role_name)
+
+    def test_model_can_create_a_role(self):
+        old_count = Role.objects.count()
+        self.role.save()
+        new_count = Role.objects.count()
+        self.assertEqual(new_count, old_count + 1)
```

Comment:
Awesome PR @andela-angene is there a reason why you are only asserting that the number of roles have increased by +1, and not checking the name of the new role that was created.